mobile_packet_verifier/src/daemon.rs

```diff
@@ -46,6 +47,7 @@ impl<S, GIR, AV> Daemon<S, GIR, AV> {
             burner,
             reports,
             burn_period: Duration::from_secs(60 * 60 * settings.burn_period as u64),
+            min_burn_period: Duration::from_secs(60 * settings.min_burn_period as u64),
```

Comment:
the Duration usage could be moved out to the settings.rs via a helper function to return the final duration.  See the existing helper function `purger_interval` in settings.rs  as an example